scripts/create-tables.sql

```diff
@@ -30,8 +29,42 @@ CREATE TABLE IF NOT EXISTS "projects"
   CONSTRAINT projects_id_pk PRIMARY KEY (id)
 );
 
-/* Triggers */
+CREATE TABLE IF NOT EXISTS "pages"
+(
+  id bigserial NOT NULL,
+  project_id bigint REFERENCES projects(id),
+  x integer NOT NULL,
+  y integer NOT NULL,
+  created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  deleted_at timestamp DEFAULT NULL,
+  styles jsonb NOT NULL DEFAULT '{}'::JSONB,
+  CONSTRAINT pages_id_pk PRIMARY KEY (id),
+  UNIQUE (project_id, x, y)
+);
 
+CREATE TABLE IF NOT EXISTS "elements"
+(
+  id bigserial NOT NULL,
+  type varchar NOT NULL,
+  page_id bigint REFERENCES pages(id),
+  created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  deleted_at timestamp DEFAULT NULL,
+  attributes jsonb NOT NULL DEFAULT '{}'::JSONB,
+  styles jsonb NOT NULL DEFAULT '{}'::JSONB,
+  CONSTRAINT elements_id_pk PRIMARY KEY (id)
+);
+
+/* Indexes */
+CREATE INDEX user_idx_id_deleted_at ON users (id, deleted_at);
+CREATE INDEX project_id_user_id_deleted_at ON projects (id, user_id, deleted_at);
+CREATE INDEX project_deleted_at_user_id on projects (deleted_at, user_id);
+CREATE INDEX project_id_deleted_at ON pages (project_id, deleted_at);
+CREATE INDEX pages_id_x_y_deleted_at ON pages (id, x, y, deleted_at);
+CREATE INDEX elements_page_is_deleted_at ON elements (id, page_id, deleted_at);
```

Comment:
Getting a staging instance up w/ a monitoring lib installed (nodetime, strongloop, newrelic, etc) is a great way to test this across all resources.